taichi/gui/cocoa.cpp

```diff
@@ -372,6 +373,14 @@ void GUI::process_event() {
           }
           break;
         }
+        case NSEventTypeScrollWheel: {
+          set_mouse_pos(p.x, p.y);
+          const auto dx = (int)cast_call<CGFloat>(event, "scrollingDeltaX");
+          const auto dy = (int)cast_call<CGFloat>(event, "scrollingDeltaY");
+          key_events.push_back(KeyEvent{KeyEvent::Type::move, "Wheel",
+                                        cursor_pos, Vector2i{dx, dy}});
```

Comment:
Is the direction correct along y? iirc mac has a reversed scroll dir than win&x11, could you run examples/keyboard.py and confirm that 'scrolling up like win' scales up the dot?

Reply:
Good point.. I flipped delta Y so that, with Mac's default settings, when swipe **up**, the deltaY will be **positive**. See

![zoom](https://user-images.githubusercontent.com/7481356/86535546-0c9fa500-bf1c-11ea-87cb-88a1e4066a17.gif)

Swiping up will increase the circle's radius. Is this consistent with Win/Linux?

Reply:
Correct √